CHANGELOG.md

```diff
@@ -25,6 +26,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Mnemonic recovery.
 - Hidden normal quitting error (acting as an install-update-and-quit instead of a simple daemon closing).
 - Missing prefixes in (crypto)currencies top shown values.
+- Payments page caret down icon.
```

Comment:
It's not a change, it's a fix ;) Changed is for stuff that worked before and has been changed.

Reply:
it's under `Fixed` already